app/views/user/attendances/show.html.erb

```diff
@@ -4,7 +4,9 @@
 <h3><%= button_to "Retake Attendance", attendance_retake_path(@attendance), class: 's-button' %></h3>
```

Comment:
I like the further separation of each partial pages functionality, and then displaying each of those pages indicidually within the attendances/show page. It looks cleaner this way. Great job!